src/display/fetch_stream.js

```diff
@@ -80,6 +81,10 @@ class PDFFetchStreamReader {
       this._disableRange = true;
     }
 
+    if (window.AbortController) {
```

Comment:
Drive-by nit: It's probably best to *not* reference `window` in `/src` code unless actually necessary, so `typeof AbortController !== 'undefined'` or `typeof AbortController === 'function'` would seem like a much better way of feature-testing here.

Reply:
Seems reasonable. Done in new commit. Thanks.